src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -49,7 +49,13 @@ abstract class AbstractObjectNormalizer extends AbstractNormalizer
      * @var callable|null
      */
     private $maxDepthHandler;
-    private $objectClassResolver;
+
+    /**
+     * @var callable|null
+     *
+     * @internal
+     */
+    protected $objectClassResolver;
```

Comment:
no need to make it protected, private would be just fine

Reply:
private on `ObjectNormalizer` child class I mean of course :)

Reply:
Fixed